File: daro-core-m/ios/Classes/DaroCoreMPlugin.swift
import Flutter
import UIKit

public class DaroCoreMPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    // Reward 앱용 DARO SDK 의존성만 포함
    // 실제 SDK 초기화는 flutter_daro_sdk 플러그인에서 처리
  }
}

File: core/daro-core-m/android/src/main/kotlin/com/example/daro_core_m/DaroCoreMPlugin.kt
package com.example.daro_core_m

import io.flutter.embedding.engine.plugins.FlutterPlugin

/** DaroCoreMPlugin - Reward 앱용 DARO SDK Core 플러그인 */
class DaroCoreMPlugin: FlutterPlugin {
  override fun onAttachedToEngine(flutterPluginBinding: FlutterPlugin.FlutterPluginBinding) {
    // Reward 앱용 DARO SDK 의존성만 포함
    // 실제 SDK 초기화는 flutter_daro_sdk 플러그인에서 처리
  }

  override fun onDetachedFromEngine(binding: FlutterPlugin.FlutterPluginBinding) {
    // 정리 작업
  }
}

